Packs/CommonScripts/Scripts/ParseEmailFilesV2/ParseEmailFilesV2.py

```diff
@@ -0,0 +1,126 @@
+    md += u"\n\n" + tableToMarkdown('HeadersMap', email_data.get('HeadersMap'))
+    return md
+
+
+def save_file(file_name, file_content) -> str:
+    """
+    save attachment to the war room and return the file internal path.
+
+    Args:
+      file_name (str): The name of the file to be created.
+      file_content (str/bytes): the file data.
+
+    Returns:
+      str: the file internal path
+
+    """
+    created_file = fileResult(file_name, file_content)
+    file_id = created_file.get('FileID')
+    attachment_internal_path = demisto.investigation().get('id') + '_' + file_id
+    demisto.results(created_file)
+
+    return attachment_internal_path
+
+
+def main():
+    file_type = ''
+    entry_id = demisto.args()['entryid']
+    max_depth = int(demisto.args().get('max_depth', '3'))
+
+    if max_depth < 1:
+        return_error('Minimum max_depth is 1, the script will parse just the top email')
+
+    parse_only_headers = demisto.args().get('parse_only_headers', 'false').lower() == 'true'
+    try:
+        result = demisto.executeCommand('getFilePath', {'id': entry_id})
+        if is_error(result):
+            return_error(get_error(result))
+
+        file_path = result[0]['Contents']['path']
+        file_name = result[0]['Contents']['name']
+        result = demisto.executeCommand('getEntry', {'id': entry_id})
+        if is_error(result):
+            return_error(get_error(result))
+
+        file_metadata = result[0]['FileMetadata']
+        file_type = file_metadata.get('info', '') or file_metadata.get('type', '')
+
+    except Exception as ex:
+        return_error(
+            "Failed to load file entry with entry id: {}. Error: {}".format(
+                entry_id, str(ex) + "\n\nTrace:\n" + traceback.format_exc()))
+
+    try:
+        email_parser = EmailParser(file_path=file_path, max_depth=max_depth, parse_only_headers=parse_only_headers,
+                                   file_info=file_type)
+        output = email_parser.parse()
+
+        resultss = []
+        if isinstance(output, dict):
+            output = [output]
+
+        for email in output:
+            if email.get('AttachmentsData'):
+                for attachment in email.get('AttachmentsData'):
+                    if attachment.get('Name') and attachment.get('FileData'):
+                        content = attachment.get('FileData')
+                        del attachment['FileData']
+                        name = attachment.get('Name')
+                        attachment['FilePath'] = save_file(name, content)
+            resultss.append(CommandResults(outputs_prefix='Email',
+                            outputs=email,
+                            readable_output=data_to_md(email, file_name, email.get('ParentFileName', None),
+                                                       print_only_headers=parse_only_headers),
+                            raw_response=email,
+                            entry_type=EntryType.NOTE))
```

Comment:
You can remove this, the default is note
```suggestion
)
```